pegasus/sites.v3/code.org/public/index.haml

```diff
@@ -29,7 +29,8 @@ style_min: true
   - if request.language == "en"
     = view :curriculum_banner
     = view :homepage_below_hero_announcement
-    = view :professional_learning_marketing_banner, audience_text: 'you', link_address: '/educate/professional-learning/middle-high', button_label: 'Learn more', homepage: true
+    -# Hiding the PL skinny banner for now since we'll have a PL hero banner for 2023
```

Comment:
@dmcavoy is there a better or more logic-driven way to hide this — or do you think it's ok to remove entirely? We'll be able to add it back easily if we need to.

Reply:
The old airplane banner we want to remove entirely. I think thats the one you are asking about right?

Reply:
Nope, this is the superhero banner; I think the airplane banner was already removed in a previous PR.

Reply:
I'm not sure I understand what you are hiding. It looks like you are hiding the superhero banner but the comment seems to suggest otherwise. Overall I don't think there is a great logic driven way to hide and show the PL banners unfortunately because the DCDO flag for applications is flipped on earlier than we have wanted to advertise that applications are open and then we usually take the banner down before applications close.

Reply:
This is the same superhero skinny banner partial being used on all of the other pages, but since we're doing a large hero banner here on the homepage we need to hide this one instance of the skinny banner. If we weren't doing a hero banner on the homepage, this line that I commented out would show up as expected as the superhero skinny banner. 

The skinny banner on the rest of the pages are hidden/shown with logic here in this same PR: https://github.com/code-dot-org/code-dot-org/blob/b36b78f29d8ba033be32540ed58c80fcb9108b2d/pegasus/sites.v3/code.org/views/professional_learning_marketing_banner.haml#L1

Let me know if you want to hop on a quick call if this doesn't make sense!

Reply:
Thank you for explaining that. Sorry my brain was not working right before break. I think we can remove the skinny banner on the homepage since we will use the big banner. If we need it we can always look up the PR.

Reply:
Haha no worries at all! 😄